### ISSUE TITLE
Missing headers for `curl` code export

### ISSUE THREAD

**Issue by filfreire** [a85/POSTMan-Chrome-Extension#8550]
**Describe the bug**
When exporting a curl request, all the headers set up are missing, instead we only get:
```
--header 'Content-Type: text/plain' \
```

**To Reproduce**
Setup a request with a few headers. Try to export the request with curl.

**Screenshots**
![screen1](https://user-images.githubusercontent.com/11976836/83010067-fc8ccf80-a017-11ea-8a08-bc688853422e.jpg)
![screen2](https://user-images.githubusercontent.com/11976836/83010071-feef2980-a017-11ea-911e-6e9c4d86ca86.jpg)

**App information (please complete the following information):**
 - App Type `Native`
 - Postman Version `Version 7.25.0 (7.25.0)`
 - OS: `macOS 10.14.6`

**Comment by DannyDainton** [comment]
Hey @filfreire,

I created a dummy request on the v7.25.0 APP and I'm not seeing the same issue:

![Screenshot 2020-05-27 at 12 00 35](https://user-images.githubusercontent.com/17089546/83011714-4377c680-a012-11ea-909a-042a4f43c448.png)

Are you able to expand on the details of that request in view or create an example that you could share, just so that we can reproduce the issue?

What settings for the code generator do you have enabled?

There was a reported issue here, for a different header - https://github.com/postmanlabs/postman-app-support/issues/5403

I hate to say it but have you tried turning it off and on again.... :D

**Comment by oysteinr** [comment]
I had the same issue, but fixed it using this sequence: 
Disable the headers by clicking their checkboxes. Click "Code". Go back and enable headers. Click "Code" again, voila.

**Comment by DannyDainton** [comment]
Thanks, @oysteinr.

I was just going to suggest the same thing :D 

I can reproduce it with _some_ collections, I'm going to have another look at this to narrow down the steps to reproduce the issue.

**Comment by filfreire** [comment]
Hey @DannyDainton 

So I tried:
1) turning it on and off again - didn't work
2) did a new dummy collection - weirdly enough it returns the headers in the curl
3) my original collection - I still have the same problem 


I copy pasted and edited the request, and for two seemingly exact same requests (one created "a-new" and another copy pasted and edited, the edited one of them i see `text/plain` only header, and the newly created one i see the proper headers in the curl.

Sharing a link, maybe it'll help you debug this:
https://www.postman.com/collections/11fafc352089a833ed99

It's super weird - it happens on some requests, but on others it doesn't, even inside the same collection.

**Comment by DannyDainton** [comment]
The workaround @oysteinr - worked for me when creating the code snippets from your requests. 

Not an ideal solution though but it might unblock you for now?

**Comment by filfreire** [comment]
The workaround by @oysteinr also worked for me. Thanks 👍

**Comment by shreys7** [comment]
Thanks for reporting this @filfreire We have identified the root cause and working on a fix for it. Will keep everyone updated here.

**Comment by vunguyen18** [comment]
> I had the same issue, but fixed it using this sequence:
> Disable the headers by clicking their checkboxes. Click "Code". Go back and enable headers. Click "Code" again, voila.

thank you!!! This fixed the bug and now my code snippets are back to normal

**Comment by mdunn-bluefox** [comment]
> I had the same issue, but fixed it using this sequence:
> Disable the headers by clicking their checkboxes. Click "Code". Go back and enable headers. Click "Code" again, voila.

This workaround solved my problem.

**Comment by mccannt** [comment]
Closing this issue as it has now been resolved.
Please feel free to reopen with the steps to reproduce if the issue persists.

**Comment by mdunn-bluefox** [comment]
Verified fixed with 7.26

**Comment by sdzialowski-godaddy** [comment]
[oysteinr](https://github.com/oysteinr) - https://github.com/postmanlabs/postman-app-support/issues/8550#issuecomment-634604589
```
I had the same issue, but fixed it using this sequence:
Disable the headers by clicking their checkboxes. Click "Code". 
Go back and enable headers. Click "Code" again, voila.
```

what do you mean >Click "Code<?
